Formula/mercury.rb

```diff
@@ -1,8 +1,8 @@
 class Mercury < Formula
   desc "Logic/functional programming language"
   homepage "https://mercurylang.org/"
-  url "https://dl.mercurylang.org/release/mercury-srcdist-20.01.2.tar.gz"
-  sha256 "95e0aa596a6437f08d7323f032dcf8356a6ce53ff88d15eb93583daea7a19520"
+  url "http://dl.mercurylang.org/release/mercury-srcdist-20.06.tar.gz"
```

Comment:
Can this go back to HTTPS, please? It seems to work for me.

```suggestion
  url "https://dl.mercurylang.org/release/mercury-srcdist-20.06.tar.gz"
```

Reply:
Oops, sorry, missed that bit while copying the URL from the Mercury site.